### PR TITLE
feature/admin-can-export-orders-to-CSV

### PR DESCRIPTION
**Pull request by rniedziolka**
Hey,
Admin can export orders to CSV file:
https://www.loom.com/share/8f1597f5d1ca46609c661788d2b0b226
Please let me know what you think about this solution!

### CODE REVIEW

File: app/services/export_service/order_export.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+require 'csv'
+
+module ExportService
+  class OrderExport
+    def initialize(data)
+      @data = data
+    end
+
+    def to_csv
+      attributes = %w[order_title order_description employee_email transaction_price created_at status]
```
Comment (oskarlakner) [inline app/services/export_service/order_export.rb]: you're missing an updated_at

File: app/services/export_service/order_export.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module ExportService
```
Comment (oskarlakner) [inline app/services/export_service/order_export.rb]: it should be called without `Service`. IMO it's not a good practice

Comment (rniedziolka) [reply app/services/export_service/order_export.rb]: I have fixed it.

File: app/services/export_service/order_export.rb (hunk 1 above)
Comment (rniedziolka) [reply app/services/export_service/order_export.rb]: I have fixed it.

**Comment by rniedziolka** [comment]
Hey @oskarlakner,
My fixes added: https://github.com/rniedziolka/employee-recognition-platform-1637255604/pull/33/commits/c104c0e72ad2be2889bc5df031ce872a4d805e80
Best,
Robert

**Review by oskarlakner — APPROVED** [review]
🙌

**Pull request merged**